cfgov/legacy/templates/front/base_update.html

```diff
@@ -174,25 +183,27 @@
 <!-- for search.usa.gov indexing -->
 <script type="text/javascript">
   //<![CDATA[
-    var aid = "cfpb";
-    var script = document.createElement('script');
-    script.type = "text/javascript";
-    script.src = "http://search.usa.gov/javascripts/stats.js";
-    document.getElementsByTagName("head")[0].appendChild(script);
+    var aid = 'cfpb';
+    var script = document.createElement( 'script' );
+    script.type = 'text/javascript';
+    script.src = 'http://search.usa.gov/javascripts/stats.js';
+    document.getElementsByTagName( 'head' )[0].appendChild( script );
   //]]>
 </script>
 
 <script type="text/javascript">
     // If there are no breadcrumbs on this page, remove the ul tag so that screen
     // readers don't read it (since the ul has nothing in it).
     $(function(){
-        var numBreadCrumbs = $("ul.bread").children("li").length;
-        if(numBreadCrumbs == 0){
-            $("ul.bread").remove();
+        var numBreadCrumbs = $( 'ul.bread' ).children( 'li' ).length;
```

Comment:
I don't see why not, but prob best to test the removal and I have no idea how to test this template. I just learned this was used on paying for college, askcfpb, and complaint and it would be helpful if it followed the base.html template that the rest of the site uses.